src/v2/components/Authentication.jsx

```diff
@@ -51,10 +53,14 @@ export default function Authentication(props) {
 
   const handleCredentials = useCallback((event) => {
     event.preventDefault()
-    query
+    setMessage('')
+
+    setTimeout(() => query
       .post('/api/loginPassword', {username, password})
       .then(onAuthenticated)
-      .catch(() => setMessage('Incorrect username or password'))
+      .catch(() => {
+        setMessage('Incorrect username or password')
+      }), 250)
```

Comment:
To allow the user time to perceive that the existing error message is hidden before displaying another error message. Without the delay, the response form the server may be so rapid that it will look like nothing happened when the for submission results in the same error. I will add a comment to this effect.

Reply:
Done